src/box/iproto_constants.h

```diff
@@ -179,6 +179,17 @@ enum iproto_key {
 	 * authentication method.
 	 */
 	IPROTO_AUTH_TYPE = 0x5b,
+	/**
+	 * Space name used instead of identifier (IPROTO_SPACE_ID) in CRUD
+	 * requests. Ignored when identifier is present.
+	 */
+	IPROTO_SPACE_NAME = 0x5c,
+	/**
+	 * Index name used instead of identifier (IPROTO_INDEX_ID) in
+	 * IPROTO_SELECT and IPROTO_DELETE requests. Ignored when identifier is
```

Comment:
Nit: It also may be used in update requests. Same note to the comment to `IPROTO_FEATURE_SPACE_AND_INDEX_NAMES`.

Reply:
Thanks, I missed it, when writing the design document! Fixed.